src/master-playlist-controller.js

```diff
@@ -268,7 +268,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       if (event.data.source === 'main') {
         this.mainSegmentLoader_.handleDecrypted_(event.data);
       } else if (event.data.source === 'audio') {
-        this.audiosegmentloader_.handleDecrypted_(event.data);
+        this.audioSegmentLoader_.handleDecrypted_(event.data);
```

Comment:
There should be more than one, such as
https://github.com/videojs/videojs-contrib-hls/blob/master/src/master-playlist-controller.js#L701